app/views/steps/miam_exemptions/adr/edit.html.erb

```diff
@@ -1,34 +1,29 @@
 <% title t('.page_title') %>
+<% step_header %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <%= step_header %>
-    <p class="app__section_heading"><%=t '.section' %></p>
-
-    <h1 class="app__heading heading-xlarge gv-u-heading-xxlarge"><%=t '.heading' %></h1>
-
-    <div class="miam_exemptions or-block__shift">
-    <%= step_form @form_object do |f| %>
-      <%=
-        f.check_box_fieldset :adr_exemptions, [
-          AdrExemptions::GROUP_PREVIOUS_ADR,
-          AdrExemptions::GROUP_PREVIOUS_MIAM
-        ] do |fieldset|
-          fieldset.check_box_input(AdrExemptions::GROUP_PREVIOUS_ADR) {
-            f.check_box_fieldset :exemptions_group, AdrExemptions::PREVIOUS_ADR
-          }
-          fieldset.check_box_input(AdrExemptions::GROUP_PREVIOUS_MIAM) {
-            f.check_box_fieldset :exemptions_group, AdrExemptions::PREVIOUS_MIAM
-          }
-        end
-      %>
-
-      <p class="form-block or-block"><%=t '.or' %></p>
-
-      <%= f.single_check_box AdrExemptions::ADR_NONE %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary %>
+
+    <span class="govuk-caption-xl"><%=t '.section' %></span>
+    <h1 class="govuk-heading-xl"><%=t '.heading' %></h1>
+
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_check_boxes_fieldset :adr, legend: { tag: 'span', size: 'm' } do %>
+
+        <%= f.govuk_check_box :adr, AdrExemptions::PREVIOUS_ADR[0].to_s, link_errors: true %>
+        <%= f.govuk_check_box :adr, AdrExemptions::PREVIOUS_ADR[1].to_s %>
```

Comment:
Not a blocker, if you don't care about their positions after the first one, you could do something similar to

```ruby
<% check_values = AdrExemptions::PREVIOUS_ADR[1..2] + AdrExemptions::PREVIOUS_MIAM %>

<% check_values.each do |value| %>
   <%= f.govuk_check_box :adr, value.to_s %>
<% end %>
```

Reply:
Yep! it makes sense in this case. Will change it.